src/utils/index.coffee

```diff
@@ -203,7 +217,15 @@ findKnownImportInDirectory = ({
     allowed
     context
   }
-  return null unless found = directoryCache.get name
+
+  return null unless (
+    found = directoryCache.get(
```

Comment:
This seems too broad currently - this will also lowercase *named* exports (rather than just *filename*/default exports)

So I'd think (since the named and filename imports are mixed together in `directoryCache`) the logic would have to be something like:
```
foundExact = directoryCache.get(name)
foundCaseInsensitive =
  if settings['known-imports/case-insensitive']
    directoryCache.get(name.toLowerCase())
return null unless (found = foundExact or foundCaseInsensitive)
{..., type} = found
return null if type isnt 'filename' and not foundExact
...
```
And correspondingly add a test (that should be failing at the moment) where you check that an import isn't generated for a case-insensitive match of a named export

Or could refactor to store two separate directory caches for filename vs named exports, I suppose?

Reply:
Fixed.